pykechain/models/representations/representations.py

```diff
@@ -29,6 +29,25 @@ class SignificantDigits(DecimalPlaces):
     rtype = PropertyRepresentation.SIGNIFICANT_DIGITS
 
 
+class ThousandsSeparator(BaseRepresentation):
+    """Representation for integer or floating-point value properties."""
+
+    rtype = PropertyRepresentation.THOUSANDS_SEPARATOR
+
```

Comment:
There is no _config_value_key required for this representation type? (this is taken from the back-end JSON)

Reply:
No _config_value_key for this representation type indeed